templates/datarobot-java/model_manifest.json.freeze

```diff
@@ -0,0 +1 @@
+{"required_files": [{"name": "sentiment_model", "source_uri": "data://demo/datarobot_models/618e9e830d2eca5ba9cd1014.jar", "md5_checksum": "6ebc07d2f1f0edffef1bc6aff24c7b93"}], "optional_files": [], "timestamp": "1636737821.9979346", "lock_checksum": "80d2a2056950f038eea78c8bb6f8c8a2"}
```

Comment:
I'm not sure I like having a pre-frozen file in the template

Reply:
it also links to a model file that won't necessarily move with the environment (at least not automatically) between clusters. Not sure the best path here but the DR Template doesn't do much without a model

Reply:
Without a frozen template this algorithm wouldn't be able to compile using the web editor or run without the `algo freeze`. Alternatively we could add an `algo freeze` operation to the build process via the web editor

Reply:
How about modifying the ADK to not require a ".freeze" file? The user may have a model_manifest.json in their algo repo, but maybe they haven't frozen this yet? They may be still experimenting with the algo and unless the algo is ready to be published, a frozen manifest might not be needed in practice.
If there's a .freeze file though, we can infer that this user wants to protect the model and in that case ADK can check the hashes, etc.

Reply:
that's an interesting idea, if there's no freeze file we just download the .json data and don't check for version conflicts.